CONTRIBUTING.md

```diff
@@ -73,6 +73,8 @@ docs
 
 Your PR should *always* target the master branch. You should *never* open a PR that targets the live branch.
```

Comment:
@BillWagner how would you feel about a rewrite like this instead?

```suggestion
Your PR should *always* target the default branch of the repository (unless you're working on a release branch). For dotnet/docs, master branch is the default branch. For the localized repositories, live branch is the default one. You should *never* open a PR that targets the live branch on dotnet/docs.
```

Reply:
@mairaw, This rewrite LGTM, waiting @BillWagner feedback before applying it.

Reply:
I agree with @mairaw 's suggestion.